entity/ProductStoreEntities.xml

```diff
@@ -75,6 +75,9 @@ along with this software (see the LICENSE.md file). If not, see
             <key-map field-name="returnPostalContactMechId"/></relationship>
 
         <relationship type="one" related="moqui.resource.wiki.WikiSpace" short-alias="wikiSpace"/>
+        <relationship type="many" related="mantle.product.store.ProductStoreDataDocument" short-alias="storeDataDocuments">
+            <key-map field-name="productStoreId"/></relationship>
+        <!-- TODO: dataDocumentId fields are to be depricated -->
```

Comment:
This comment about deprecation might be best moved up above the 3 *DataDocumentId fields, ie before the current line 49. Because not everyone looks at the XML files comments are not always seen, so also better to put this in a description for each field. The last bit for deprecation comments is to mentioned what replaces it, in this case the new ProductStoreDataDocument entity that you added.

Reply:
Should be done. Check update.